src/library_dylink.js

```diff
@@ -952,7 +970,7 @@ var LibraryDylink = {
   },
 
   _dlopen_js__deps: ['$dlopenInternal'],
-  _dlopen_js__sig: 'iiii',
```

Comment:
I don't quite undersand why the number of arguments here and below is different

Reply:
You mean the difference between `dlopen_js` and `emscirpten_dlopen_js`?   The former is synchronous and returns that handle.  The later is asynchronous (and void) and takes onsuccess/onerror handlers.

Reply:
I was confused about why the signature here changes from 4 things `iiii` to two things `pp`. Looks like it was just wrong before, and should always have been 2?

Reply:
Yeah, that's what I meant as well. If it was wrong before that makes sense (I guess if there were extra args they would have been ignore?)

Reply:
Yes they were just plain wrong before.